woocommerce/woocommerce-ios
# Update Order Details to match i3 designs

This PR is a pre-requisite to #1099. Per the i3 designs found in Figma, the Order Details page has been updated with some minor visual changes.

### Changes
- Re-order the files found inside of the Order folder (see table 1 below)
- Remove the "Paid by Customer" subview inside of the payment tableview cell (see table 2 below)
- Add "Paid by Customer" as its own tableview cell (see table 2 below)

## To test
- Check out this branch, build and run
- Navigate to any order details page and observe: border line above "Paid by Customer" cell has changed, "Paid By Customer" is in Title Case instead of sentence case, and smaller text is now set to Footnote size.

| Orders | Order Details expanded |
| --- | --- |
| <img width="265" alt="Screen Shot 2019-10-01 at 3 17 20 PM" src="https://user-images.githubusercontent.com/1062444/65997785-a3a47180-e45f-11e9-99f6-edf162e1332c.png"> | <img width="334" alt="Screen Shot 2019-10-01 at 3 25 58 PM" src="https://user-images.githubusercontent.com/1062444/65997846-c767b780-e45f-11e9-9a65-ad77e2343bb1.png"> |

![update-to-order-details](https://user-images.githubusercontent.com/1062444/66086413-a6739500-e539-11e9-9f59-96ca5de02adf.gif)

| Before | After |
| --- | --- |
| <kbd><img src="https://user-images.githubusercontent.com/1062444/66086444-b55a4780-e539-11e9-995b-f05061402940.png" width="350" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/1062444/66086462-c7d48100-e539-11e9-9063-3f1bebf27d4e.png" width="350" /></kbd> |

Update release notes:
